website/docs/d/workspaces_bundle.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_workspaces_bundle"
+sidebar_current: "docs-aws-datasource-workspaces-bundle"
+description: |-  
+  Get information on a Workspaces Bundle.
+---
+
+# aws_workspaces_bundle
```

Comment:
Most likely a change that occurred after the creation of this pull request, but can you please prepend the title with `Data Source: `, e.g. `Data Source: aws_workspaces_bundle`